include/glfw/window/window.h

```diff
@@ -131,6 +132,12 @@ class Window
     // Call permissions: main thread.
     void setFullscreen() const;
 
+    // Call permissions: any thread.
+    static vk::SurfaceKHR createWindowSurface(
```

Comment:
Change ```vk::SurfaceKHR``` to an owning wrapper ```vk::UniqueSurfaceKHR``` to make the code correct by default.

Reply:
Otherwise it is possible to leak the surface resource if one forgets to call ```vk::SurfaceKHR::destroy()```

Reply:
Window system doesn't care about it. This is your point how to handle.